### ISSUE TITLE
Dockerhub automated builds are no more available for free accounts

### ISSUE THREAD

**Issue by znerol** [prometheus-pve/prometheus-pve-exporter#80]
Automated builds now requires paid pro or team account. Applied as an Open Source project, let's see whether we qualify.

**Comment by michaelarnauts** [comment]
You could (in the meantime) push manually if you like.
```
docker login --username <username>
docker build -t <hub-user>/<repo-name>[:<tag>]
docker push <hub-user>/<repo-name>:<tag>
```

**Comment by znerol** [comment]
> You could (in the meantime) push manually if you like.

Right. I prefer to avoid that though.

**Comment by ilude** [comment]
what about using Github actions and publishing to Github Packages instead?

**Comment by znerol** [comment]
New image with the following tags is now available on dockerhub: `latest`, `2`, `2.2`, `2.2.2`. Thanks @ilude for the help.
